src/binary.ts

```diff
@@ -485,6 +471,26 @@ export class UUID extends Binary {
     return new UUID(ByteUtils.fromBase64(base64));
   }
 
+  /** @internal */
+  static uuidBytesFromString(representation: string) {
```

Comment:
The naming is redundant here since it's a static `UUID` method
```suggestion
  static bytesFromString(representation: string) {
```

Reply:
SGTM